postgresql/gen-yaml/gen.sh

```diff
@@ -0,0 +1,27 @@
+
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o nounset
+
+mkdir -p manifests/upstream
+rm manifests/upstream/*
+
+# Fetch from helm repo: https://github.com/bitnami/charts/tree/main/bitnami/postgresql
+helm repo add bitnami https://charts.bitnami.com/bitnami
+
+helm template "postgresql" bitnami/postgresql --version "${BITNAMI_POSTGRES_RELEASE}" \
```

Comment:
It would be cool to enable also the metrics exporter with `metrics.enabled=true`: https://github.com/bitnami/charts/tree/main/bitnami/postgresql#metrics-parameters

Reply:
I might need to spend some more time on that. It looks like if you set the metrics you also need to pass in a database name to helm. Which is then hardcoded into some of the other parts that might be hard to change in kustomize